### PR TITLE
Improve path slashes of authentication imports

### PR DESCRIPTION
**Pull request by nirvn**
Tiny cleanup of / slashes in the authentication imports code (I don't think it was consequential). I've also added a debug message to help debugging.

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for ac1c249ebda13e3d7dab87120912147b4fd02583:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-46f45ae6c1601000d2b9b4bc11d54f3d2945759b-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-46f45ae6c1601000d2b9b4bc11d54f3d2945759b-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-46f45ae6c1601000d2b9b4bc11d54f3d2945759b-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-46f45ae6c1601000d2b9b4bc11d54f3d2945759b-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for a215378740def5307e49454e796bc868903f1b78:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-321b5ede894be981e7d3729f6d5f08b6bd053faf-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-321b5ede894be981e7d3729f6d5f08b6bd053faf-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-321b5ede894be981e7d3729f6d5f08b6bd053faf-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-321b5ede894be981e7d3729f6d5f08b6bd053faf-x86-android.apk)

File: src/core/qgismobileapp.cpp [hunk 1]
```diff
@@ -300,12 +301,13 @@ QgisMobileapp::QgisMobileapp( QgsApplication *app, QObject *parent )
...
     for ( const QString &dataDir : dataDirs )
     {
-      QDir configurationsDir( QStringLiteral( "%1/auth/" ).arg( dataDir ) );
+      QDir configurationsDir( QStringLiteral( "%1/auth" ).arg( dataDir ) );
       if ( configurationsDir.exists() )
       {
         const QStringList configurations = configurationsDir.entryList( QStringList() << QStringLiteral( "*.xml" ) << QStringLiteral( "*.XML" ), QDir::Files );
         for ( const QString &configuration : configurations )
         {
+          qDebug() << QStringLiteral( "Importing authentication configuration file %1" ).arg( configurationsDir.absoluteFilePath( configuration ) );
```
Comment (suricactus) [inline src/core/qgismobileapp.cpp:310]: If you need to log, why don't you use the official `QgsLogger` API?

Comment (nirvn) [reply src/core/qgismobileapp.cpp:310]: @suricactus ,

QgsLogger -> shows up in the message log, not great.

QgsDebugMsg -> fails to print in the adb logcat

qDebug() -> :heart: 

It's not meant to end up merged, just helping out debugging an issue.

Comment (suricactus) [reply src/core/qgismobileapp.cpp:310]: If you plan to keep it, then change the include here `#include <qgslogger.h>`

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for d2a2b3edda2cd511776ec849450582befd61eba5:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-442569544d15b626b9ca21835f8bf494b8e090a1-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-442569544d15b626b9ca21835f8bf494b8e090a1-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-442569544d15b626b9ca21835f8bf494b8e090a1-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-442569544d15b626b9ca21835f8bf494b8e090a1-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for bbc7c44815b08cbce15420b0b27d7df0ab08ea3d:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-fb256a311268747bc3955195716174983d5a7778-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-fb256a311268747bc3955195716174983d5a7778-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-fb256a311268747bc3955195716174983d5a7778-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-fb256a311268747bc3955195716174983d5a7778-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 0811735b91e19c5a2a826f1be3d6f8ebaddb04f6:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-bd4fb6fbb677d190f86fa7709b073bd6829b8723-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-bd4fb6fbb677d190f86fa7709b073bd6829b8723-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-bd4fb6fbb677d190f86fa7709b073bd6829b8723-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-bd4fb6fbb677d190f86fa7709b073bd6829b8723-x86-android.apk)

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for e704a5ce9650b0f7b37c88e5284ae8c0432fec8f:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-92cc7064dbdba0a32b610889ac1a9f6a7f125856-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-92cc7064dbdba0a32b610889ac1a9f6a7f125856-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-92cc7064dbdba0a32b610889ac1a9f6a7f125856-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2923-92cc7064dbdba0a32b610889ac1a9f6a7f125856-x86-android.apk)

**Pull request closed without merging**